packages/vc-status-list-issuer-rest-api/src/api-functions.ts

```diff
@@ -96,6 +102,84 @@ export function getStatusListCredentialIndexStatusEndpoint(router: Router, conte
     console.log(`Get statusList credential index status endpoint is disabled`)
     return
   }
+
+  const path = opts?.path ?? '/status-lists/:statusListId/entry/:entryId'
```

Comment:
Keep it more similar to the other one '/status-lists/:statusListId/status/entry-by-id/:entryId

The choice to have /status in there was on purpose

Reply:
adjusted